businesscentral-monitoring/scripts/businesscentral-monitoring/install

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+SOURCES_DIR=/tmp/artifacts
+
+unzip ${SOURCES_DIR}/jboss-bpmsuite-7.0.0-business-central-monitoring-eap7.zip -d ${SOURCES_DIR}
+chmod 0755 ${SOURCES_DIR}/jboss-eap-7.0
+
+cp -rf ${SOURCES_DIR}/jboss-eap-7.0/* ${JBOSS_HOME}/
+mv "${JBOSS_HOME}/standalone/deployments/business-central-monitoring.war" "${JBOSS_HOME}/standalone/deployments/ROOT.war"
```

Comment:
@rbarriuso regarding the war final name I'm not sure it will have the _-monitoring_, I guess we can leave it out for now.